src/improvements/tasks/MultisigTask.sol

```diff
@@ -927,7 +927,7 @@ abstract contract MultisigTask is Test, Script {
     /// check the state changes applied by the task. This function can check
     /// that only the nonce changed in the parent multisig when executing a task
     /// by checking the slot and address where the slot changed.
-    function checkStateDiff(VmSafe.AccountAccess[] memory accountAccesses) internal view virtual {
+    function checkStateDiff(VmSafe.AccountAccess[] memory accountAccesses) internal virtual {
```

Comment:
Can this function not remain view? I've made the change locally and it seems to work.

Reply:
I don't think we want this to be virtual either. Templates won't be overriding this functionality. It should be run all the time regardless.

Reply:
It is currently virtual in the legacy code, but agreed it needs to be